modules/dqt/static/schema.yml

```diff
@@ -0,0 +1,346 @@
+openapi: 3.0.1
+info:
+  title: LORIS - REST API endpoints
+  description: The LORIS API uses standard HTTP error codes and the body of any response will either be empty or contain only a JSON object for any request.
+  contact:
+    name: LORIS Development Team
+    url: 'https://github.com/aces/loris'
+  license:
+    name: 'GNU Public License, Version 3'
+    url: 'https://opensource.org/licenses/GPL-3.0'
+  version: 0.0.3-dev
+servers:
+  - url: /api/v0.0.3-dev
+paths:
+ 
+  /dqt/queries:
+    get:
+      description: List available queries
+      responses:
+        '200':
+          description: All the queries in 3 lists ; History, Saved, Shared
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Queries'
+    post:
```

Comment:
When I put this into https://editor.swagger.io/ the examples that it shows have a structure such as

```
     "fields": {
        "category": "string",
        "field": "string",
        "visits": [
          "string"
        ]
      },
```

I don't know enough about swagger to know where the bug is, but I don't see how that can handle a query that has more than 1 field.

Reply:
Unsure if helpful but therels also allVisits that get attached like example from console.log:
```
{
  "specimens": {
    "specimen_list_details": {
      "Visit 02": "Visit 02"
    },
    "allVisits": {
      "Visit 02": 1,
      "Visit 01": 0,
      "ALS-TAL-V01": 0,
      "ALS-TAL-V02": 0,
      "ALS-TAL-V03": 0,
      "ALS-TAL-V04": 0,
      "ALS-TAL-VET": 0,
      "ALS-TAL-VOP": 0,
      "ALS-TAL-VT": 0,
      "BENEFIT-Bsl": 0,
      "BENEFIT-Scr": 0,
      "BENEFIT-V01": 0,
      "CEF-Base": 0,
      "CEF-Scr": 0,
      "CEF-W16": 0,
      "CEF-W24": 0,
      "EMP-Base": 0,
      "EMP-M03": 0,
      "EMP-M07": 0,
      "EMP-M08": 0,
      "EMP-M10": 0,
      "EMP-M11": 0,
      "EMP-M12": 0,
      "EMP-Scr": 0,
      "ENV-Base": 0,
      "ENV-EOT": 0,
      "ENV-F/up": 0,
      "ENV-M01": 0,
      "ENV-M02": 0,
      "Genetics-V01": 0,
      "GSK-NOG Bsl": 0,
      "GSK-NOG EW": 0,
      "GSK-NOG FUP": 0,
      "GSK-NOG Scr": 0,
      "GSK-NOG W02": 0,
      "GSK-NOG W03": 0,
      "GSK-NOG W04": 0,
      "GSK-NOG W06": 0,
      "GSK-NOG W10": 0,
      "GSK-NOG W12": 0,
      "GSK-NOG W16": 0,
      "GSK-NOG W20": 0,
      "GSK-NOG W24": 0,
      "GSK-NOG W28": 0,
      "GSK-NOG W32": 0,
      "GSK-NOG W36": 0,
      "GSK-NOG W40": 0,
      "GSK-NOG W44": 0,
      "GSK-NOG W48": 0,
      "LIT-Base": 0,
      "MRI01": 0,
      "MRI01Anonymized": 0,
      "NeuropsychologyEvaluation01": 0,
      "TOSI-ALS-V01": 0,
      "TOSI-MS-V01": 0,
      "V01 (Pre-Inf)": 0,
      "V01-Sal": 0,
      "V02 (D9 Post)": 0,
      "V02-Sal": 0,
      "V03 (D14 Post)": 0,
      "V04 (Pre-Inf)": 0,
      "Visit 01 - NDP": 0,
      "Visit 01 - SDP": 0
    }
  }
}
```